server/controllers/authController.api.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const db = require('../config/db.config.js');
+const User = db.user;
+const jwt = require('jsonwebtoken');
+const passport = require('passport')
+const bcrypt = require('bcrypt');
+const saltRounds = 12;
+const JWTSecret = process.env.JWT_SECRET || 'some secret';
+
+
+exports.register = (req, res) => {
+    const username = req.body.username;
+    const email = req.body.email;
+    const password = req.body.password;
+
```

Comment:
can same some lines by destructuring with const { username, email, password } = req.body;

Reply:
Yes thanks. I have destructing and prettification to do all over the place.